_commands/tools/sftp.md

```diff
@@ -3,22 +3,152 @@
 
 sftp
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
-
-<!-- minimal example -->
+`sftp` is used to securely transfer files between a local machine and a remote machine that is accessed via SSH. SFTP stands for Secure File Transfer Protocol.
 ~~~ bash
-*short* example of command usage and output
+$ sftp user@hostname
+Password: *********
+Connected to hostname.
+sftp> put filename.txt
+sftp> mput file1.txt file2.txt file3.txt
+sftp> get file.txt
+sftp> mget file1.txt file2.txt file3.txt
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `sftp user@hostname`
+~~~ bash
+$ sftp mmdarden@login.engin.umich.edu
+Password: **********
+Connected to login.engin.umich.edu.
+sftp>
+~~~
+
+##### Break it down
+
+ * This is how you begin the sftp process. It is done the same way as when you SSH into a machine, except you use the sftp command instead of the ssh command. You must have access to the machine and provide a password. You can try this by remote accessing CAEN with your uniqname@login.engin.umich.edu and providing your kerberos password as the password.
```

Comment:
Do these need to be bullet points? In all cases it seems like there is only one bullet following the example. This could just be body text instead of a bullet point.